tools/topology/topology1/sof-tgl-max98357a-rt5682.m4

```diff
@@ -422,10 +422,10 @@ ifelse(
 		SSP_CONFIG_DATA(SSP, SPK_SSP_INDEX, 32)))',
 	CODEC, `RT1019', `
 	SSP_CONFIG(I2S, SSP_CLOCK(mclk, 19200000, codec_mclk_in),
-                SSP_CLOCK(bclk, 1536000, codec_slave),
+                SSP_CLOCK(bclk, 3072000, codec_slave),
                 SSP_CLOCK(fsync, 48000, codec_slave),
-                SSP_TDM(2, 16, 3, 3),
-                SSP_CONFIG_DATA(SSP, SPK_SSP_INDEX, 16)))',
+                SSP_TDM(2, 32, 3, 3),
+                SSP_CONFIG_DATA(SSP, SPK_SSP_INDEX, 24)))',
```

Comment:
Looks like commit message should really say we are changing the audio format to 24 bit.